stable/redmine/templates/cronjob.yaml

```diff
@@ -0,0 +1,148 @@
+{{- if .Values.mailReceiver.enabled }}
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ template "redmine.mailReceiver.fullname" . }}
+  labels:
+    app: {{ template "redmine.mailReceiver.fullname" . }}
+    chart: {{ template "redmine.chart" . }}
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
```

Comment:
The labels definition was improved on `14.0.0` on every manifest (please take a look)

Reply:
Thank you.

I hope I did it correctly.